words_dir/tweet_labels.py

```diff
@@ -10,14 +10,14 @@
 labels = {
     # 'label': 'defintion'
     # sexual orientation
-    'gay': '',
-    'lesbian': '',
-    'bisexual': '',
-    'pansexual': '',
+    'gay': 'Possessing sexual and/or emotional attraction towards members of the same sex.',
```

Comment:
Thank you for working on this! It looks v. good 👌 

Just a small suggestion: since this definition for 'gay' is basically that of 'homosexual' in line 30, would it make more sense if we tweaked the definition of 'gay' to add:

-  'possessing sexual and/or emotional attraction towards members of the same sex; preferring mostly or exclusively men as romantic or sexual partners.' I think since this term is used by many people with different sexual orientations, it would make sense to put that there along with the fact that for most gay men, this is the sexual orientation.

Reply:
Thanks for the suggestion, how about "Possessing sexual and/or emotional attraction towards members of the same sex; Especially men preferring mostly or exclusively men as romantic or sexual partners." as an additional tweak?

Reply:
Yes, this sounds good!